stdlib/3/tkinter/__init__.pyi

```diff
@@ -1,4 +1,5 @@
 import sys
+from enum import Enum
```

Comment:
`enum` is Python 3.4+ only, needs `sys.version` guard

Reply:
I don’t have such old version of Python easily available to me for testing. What do you recommend? Should I just guard my entire addition, so people using ≤3.3 don’t get the `Event` class at all?

Reply:
Good spot, but conveniently typeshed only supports Python 3.5 and later: https://github.com/python/typeshed#about

C.f. https://github.com/python/typeshed/pull/3147